src/main/java/seedu/duke/CommandList.java

```diff
@@ -122,9 +122,9 @@ public static void executeSave(String[] readArgumentTokens) {
         float adjustedAiDirection = SaverList.saverList.get(Ui.curSaver).aiDirectionAdjust(Ai.getAiDirection());
         float adjustedRange = SaverList.saverList.get(Ui.curSaver).rangeAdjust();
         testForSave(adjustedSaveDirection, adjustedAiDirection, adjustedRange);
-        boolean isSaveSuccessful = saveCheck(adjustedSaveDirection, adjustedAiDirection, adjustedRange);
-        MatchStat.updateStat(isSaveSuccessful);
-        SaverList.saverList.get(Ui.curSaver).printGoalAfterSave(isSaveSuccessful, Math.round(adjustedSaveDirection));
+        boolean isScoreGoal = goalCheck(adjustedSaveDirection, adjustedAiDirection, adjustedRange);
+        MatchStat.updateStat(isScoreGoal);
+        SaverList.saverList.get(Ui.curSaver).printGoalAfterSave(isScoreGoal, Math.round(adjustedSaveDirection));
```

Comment:
I am not sure but for printGoalAfterSave(), do you input isGoal or isSaved? I am certain that for updateStat(), you need to input isGoal.

Reply:
you're right it should be !isGoalScored